regression-test/suites/minus_except_test/minus_except.groovy

```diff
@@ -24,7 +24,6 @@ suite("minus_except") {
     sql "CREATE TABLE ${tableNameDate}(c2 date) DUPLICATE key(c2) DISTRIBUTED BY hash(c2) BUCKETS auto PROPERTIES ('replication_num' = '1')"
     sql "insert into ${tableNameVarchar} values('+06-00');"
     sql "insert into ${tableNameDate} values('1990-11-11');"
-    qt_select " SELECT c1,c1 FROM ${tableNameVarchar} MINUS SELECT c2,c2 FROM ${tableNameDate};"
```

Comment:
This test should be added back, change the result format PLS.

Reply:
> This test should be added back, change the result format PLS.

I added this test before,but it seems can't pass on the remote which reason may be due to grammar failure（the test can be tested successfully  locallly）
I wonder if is the same reason caused like  the one replied  above